FpsCheck.py

```diff
@@ -376,8 +378,9 @@ def find_invalid_well_known(self, check_sets):
                         else:
                             field_sym_difference = []
                     else:
-                        field_sym_difference = set(json_schema[field]) ^ set(
-                        curr_fps_set.relevant_fields_dict[field])
+                        field_sym_difference = set(json_schema[field] 
+                                if field in json_schema.keys() else []) ^ set(
```

Comment:
Hm, instead of handling defaults again here, WDYT about converting the JSON object into an `FpsSet` instance? That would discard irrelevant fields for us, and let this benefit from all the default fields we're setting up there. I think that would be less error-prone than manipulating the JSON directly.

Reply:
Just occurred to me - `field` might be `"ccTLDs"`, and in that case, `[]` isn't a valid value for that field, right? We'd want to use `{}` instead. Similarly, when `field` is `"primary"`, `[]` isn't valid.

But I think in general it might be easier/clearer if we didn't try to be generic over the field, and just check the four fields explicitly. Two of the four fields share the same type, but the 4 fields have 3 different types overall (`str`, `List[str]`, `Dict[str, str]`).

Reply:
I am open to converting the JSON object into an FPS instance. In terms of your comment about using `{}` vs `[]`, I don't believe that this should matter. Regardless of what the default value is - `[]`, `{}`, `""`, etc. - it's placed inside a set constructor so it will ultimately be equal to an empty set if the field is not present. Primary is already handled differently, but this is because placing a string in a set constructor does a very silly thing.

Reply:
Right, I'm not saying there's a malignant bug here - just a benign one that happens to not cause problems right now :)